index.html

```diff
@@ -3,73 +3,171 @@
+                    <img src="./assets/icon_web-development.svg" alt="Web development.">
+                    <h4>Web Development</h4>
+                    <p>Modern websites need to be responsive and fully functional at all sizes. We develop all of
+                        our sites with a mobile first approach and ensure that our code quality is clean, organized and
+                        performant.
+                        Already have a design but need it developed? We’re here to help!</p>
+                </article>
+                <article>
+                    <img src="./assets/icon_ux.svg" alt="User experience.">
+                    <h4>User Experience</h4>
+                    <p>In order to build successful products you have to understand the user. We create user personas,
+                        wireframes and prototypes to test with users. This allows us to gather feedback which helps to
+                        shape
+                        the product and can quickly eliminate potential risks – saving you time and money.
+                    </p>
+                </article>
+            </div>
+            <div class="row">
+                <article>
+                    <img src="./assets/icon_graphic-design.svg" alt="Graphic design.">
+                    <h4>Logo Design + Branding</h4>
+                    <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily
+                        recognizable
+                        logos and brand identity. The best brands stand out from the crowd by having a clear and unique
+                        point of view. We will work with you to understand your vision and help create a unique brand
+                        identity and style guide for your company.</p>
+                </article>
+                <article>
+                    <img src="./assets/icon_seo.svg" alt="S E O.">
+                    <h4>SEO</h4>
+                    <p>Do you want your site to rank higher in search results? Google is constantly changing their
+                        algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make
+                        sure
+                        your website it optimized, providing you the best opportunities for organic search exposure.
+                    </p>
+                </article>
+                <article>
+                    <img src="./assets/icon_copywriting.svg" alt="Copywriting.">
+                    <h4>Copy Writing</h4>
+                    <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique tone
+                        of
+                        voice will help you stand out from the crowd and build brand loyalty. We work with you to
+                        establish
+                        your brand bible and ensure your messaging is focused and targeted.</p>
+                </article>
+            </div>
```

Comment:
Het zou mooi zijn geweest als de articles onder elkaar zouden komen te staan als de pagina te smaller gemaakt zou worden

Reply:
Hoi! Dank je voor je reactie. Eerlijk gezegd: ik heb de media query voor mobiele weergave eigenlijk helemaal niet gedaan, want daar moet dan vreselijk veel gedupliceerde code in, wat het geheel niet mooier maakt. Ik had eigenlijk moeten beginnen met de eenvoudige mobiele weergave, en daarna pas alle extra CSS die alleen voor desktop weergave geldt. Nu moet ik alles copy-pasten naar een nieuwe MQ en dan alles wat niet van toepassing is uit die MQ verwijderen. Ik was er al aan begonnen maar dat bleek moeiljker dan ik dacht dus heb ik die hele sectie weer uitgeveegd.

Ik heb momenteel helaas geen tijd om er verder mee te gaan, maar zodra ik wat tijd vind zal ik eraan beginnen.

Hoe ben jij aan mij gekoppeld trouwens? Weet jij misschien hoe ik erachter kom aan wie ik zelf gekoppeld ben voor deze opdracht?